dev-client/src/translations/en.json

```
@@ -89,7 +89,7 @@
       "edit_title": "Site Note",
       "note_attribution": "{{createdAt}} by {{name}}",
       "confirm_removal_title": "Delete note?",
-      "confirm_removal_body": "Are you sure you want to remove this note?",
+      "confirm_removal_body": "This action can't be undone.",
```

Use `cannot` to match other messages.
```suggestion
      "confirm_removal_body": "This action cannot be undone.",
```